### ISSUE TITLE
[Website] Image Gallery: Alt is not defined for images under Image Gallery adaptive card.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6421]
### Target Platforms

NodeJS

### SDK Version

9/5/2021

### Application Name

https://adaptivecards.io/samples/ImageGallery.html

### Problem Description

[ADO 35800985](https://microsoft.visualstudio.com/OS/_workitems/edit/35800985)

**Repro steps:**
1.Open above URL on Edge Dev and Enable Narrator.
2.Navigate to Image gallery and activate it.
3.Navigate to imaged by using up and down arrow key in scan mode and verify the issue.

**Actual Result:**
Alt is not defined for images under Image Gallery adaptive card. Screen reader only announces 'Image'.

**Expected Result:**
Alt should be properly defined for the images under image gallery card, so that screen reader should understand the control. 

### Card JSON

```JSON
ImageGallery.json
```

**Comment by paulcam206** [comment]
This has been fixed.

**Comment by PankajBhojwani** [comment]
I can't repro this, I think its been fixed?

**Comment by carlos-zamora** [comment]
Ah, the issue is with the templated sample cards. I've got this one.
